tutorials/flume-capture-events.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: page
+title: Capturing Events with Flume
+---
+
+Once you have an [Events dataset][events], you can create a web application that captures session events.
+
+This example shows how you can send log information via Flume to your Hadoop database using a JSP and custom servlet running on Tomcat.
+
+[events]:{{site.baseurl}}/tutorials/create-events-dataset.html
+
+## Configuring Flume
+
+These are the steps to configure Flume to channel log information directly to the `events` dataset. You first generate the configuration information using the Kite command-line interface, copy the results, paste them in the Flume configuration file, and then restart Flume.
+
+1. In a terminal window, type `kite-dataset flume-config --channel-type memory events`.
+1. Copy the output from the terminal window.
+1. Open Cloudera Manager.
```

Comment:
The VM setup instructions don't assume you're using Cloudera Manager, so I don't think this should either. The alternative is to copy the flume config and restart:

```
sudo cp flume.conf /etc/flume-ng/conf/flume.conf
sudo /etc/init.d/flume-ng-agent restart
```